index.js

```diff
@@ -70,5 +76,52 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+
+function increaseLikes() {
+  const newLikes = parseInt(likes.textContent) + 1
+  likes.textContent = `${newLikes} Likes`
+}
+
+
+likeBtn.addEventListener("click", increaseLikes)
```

Comment:
Nice! Don't forget to update `player.likes`. Way to break up your code!

Reply:
Although it won't persist right now, we're going to learn ways to get it to persist soon :)